lib/LANraragi/Utils/Plugins.pm

```diff
@@ -131,6 +131,8 @@ sub get_plugin_parameters {
 
             #Decode it to an array for proper use
             if ($saved_config) {
+                # encode before json-decoding it in case it has UTF8 characters in plugin config
+                $saved_config = redis_encode($saved_config);
```

Comment:
Since we're doing a `redis_decode` 3 lines prior, you might as well remove it up there, so that `decode_json` uses the DB-encoded version of the string.